test/integration/targets/docker_host_facts/tasks/test_host_facts.yml

```diff
@@ -0,0 +1,235 @@
+         - 'output.docker_disk_usage is not defined'
+
+
+  - name: Get info on Docker host and list volumes
+    docker_host_facts:
+      volumes: yes
+    register: output
+
+  - name: assert reading docker host facts when docker is running and list volumes
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list[0].Name is string'
+         - 'output.docker_volumes_list[0].Mountpoint is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage is not defined'
+
+
+  - name: Get info on Docker host and list volumes with verbose output
+    docker_host_facts:
+      volumes: yes
+      verbose_output: yes
+    register: output
+
+  - name: assert reading docker host facts when docker is running and list volumes with verbose output
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list[0].Name is string'
+         - 'output.docker_volumes_list[0].Mountpoint is string'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage is not defined'
+
+
+  - name: Get info on Docker host and get disk usage
+    docker_host_facts:
+      disk_usage: yes
+    register: output
+
+  - name: assert reading docker host facts when docker is running and get disk usage
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage.LayersSize is number'
+         - 'output.docker_disk_usage.BuilderSize is not defined'
+
+
+  - name: Get info on Docker host and get disk usage with verbose output
+    docker_host_facts:
+      disk_usage: yes
+      verbose_output: yes
+    register: output
+
+  - name: assert reading docker host facts when docker is running and get disk usage with verbose output
+    assert:
+      that:
+         - 'output.docker_host_facts.Name is string'
+         - 'output.docker_containers_list is not defined'
+         - 'output.docker_networks_list is not defined'
+         - 'output.docker_volumes_list is not defined'
+         - 'output.docker_images_list is not defined'
+         - 'output.docker_disk_usage.LayersSize is number'
+         - 'output.docker_disk_usage.BuilderSize is number'
+
+  always:
+  - name: Delete container
+    docker_container:
+      image: alpine:3.8
+      command: '/bin/sh -c "sleep 10m"'
+      name: "{{ cname }}"
+      state: absent
+    register: container_output
```

Comment:
Remove the `register` here and in the next task. Also, there's a double empty line between these tasks.